contracts/TablelandTables.sol

```diff
@@ -46,28 +46,77 @@ contract TablelandTables is
     }
 
     /**
-     * @dev See {ITablelandTables-createTable}.
+     * @dev See {ITablelandTables-runSQL}.
      */
     function createTable(
         address owner,
-        string memory statement
-    ) external payable override whenNotPaused returns (uint256 tableId) {
-        tableId = _nextTokenId();
-        _safeMint(owner, 1);
+        string calldata statement
+    ) external payable override whenNotPaused returns (uint256) {
+        return _createTable(owner, statement);
+    }
 
-        emit CreateTable(owner, tableId, statement);
+    /**
+     * @dev See {ITablelandTables-writeToTable}.
+     */
+    function writeToTable(
+        address caller,
+        uint256 tableId,
+        string calldata statement
+    ) external payable override whenNotPaused nonReentrant {
+        _mutateTable(caller, tableId, statement);
+    }
 
-        return tableId;
+    /**
+     * @custom:depreciated See {ITablelandTables-runSQL}.
+     */
+    function runSQL(
+        address caller,
+        uint256 tableId,
+        string calldata statement
+    ) external payable override whenNotPaused nonReentrant {
+        _mutateTable(caller, tableId, statement);
     }
 
     /**
      * @dev See {ITablelandTables-runSQL}.
      */
     function runSQL(
         address caller,
-        uint256 tableId,
-        string memory statement
+        ITablelandTables.Runnable[] calldata runnables
     ) external payable override whenNotPaused nonReentrant {
```

Comment:
@carsonfarmer Hopefully this function can enable the new sdk's batch method to do more interesting stuff, or at least save some gas.